homework9_symbol.py

```diff
@@ -0,0 +1,37 @@
+"""homework9"""
+
+# Подсчет количества букв
+# На вход подается строка, например, "cccbba"
+# результат работы программы - строка “c3b2a"
+# Примеры для проверки работоспособности:
+# "cccbba" == "c3b2a"
+# "abeehhhhhccced" == "abe2h5c3ed"
+# "aaabbceedd" == "a3b2ce2d2"
+# "abcde" == "abcde"
+# "aaabbdefffff" == "a3b2def5"
+
+
+def count_s(a):
+    """Преобразование из "cccbba" в c3b2a"""
+    symbol = a[0]
+    count = 0
+    sum_sym = ""
+    for [_, s] in enumerate(a):
```

Comment:
since you are not using `index`, you can iterate over a string:
`
for i in a:
...
`